<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectSetupWidget</class>
 <widget class="QDialog" name="ProjectSetupWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ProjectSetupWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Project setup</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="horizontalSpacing">
       <number>6</number>
      </property>
      <item row="3" column="2">
       <widget class="QTimeEdit" name="p_timeEdit_StandartRestTime">
        <property name="dateTime">
         <datetime>
          <hour>12</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2000</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="3">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QLabel" name="p_lbl_MaxPercentOfWorkShiftToPeriod">
        <property name="text">
         <string>Maximum hours of working time per period</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="p_lbl_StandartRestTime">
        <property name="text">
         <string>Standart rest time for workers</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="p_lbl_StartOfPeriod">
        <property name="text">
         <string>Start of period</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QSpinBox" name="p_spinBox_MaxHoursPerPeriod">
        <property name="suffix">
         <string/>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>40</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="p_lbl_MinRestTime">
        <property name="text">
         <string>Minimum rest time for workers</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QTimeEdit" name="p_timeEdit_MinRestTime">
        <property name="dateTime">
         <datetime>
          <hour>10</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2000</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="p_lbl_EndOfPeriod">
        <property name="text">
         <string>End of period</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QDateTimeEdit" name="p_dateTimeEdit_StartPeriod">
        <property name="wrapping">
         <bool>true</bool>
        </property>
        <property name="calendarPopup">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QDateTimeEdit" name="p_dateTimeEdit_EndPeriod">
        <property name="wrapping">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="p_buttons_layout">
     <item>
      <widget class="QPushButton" name="p_button_Save">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>p_timeEdit_MinRestTime</tabstop>
  <tabstop>p_timeEdit_StandartRestTime</tabstop>
  <tabstop>p_button_Save</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
